doc/dynamic-environments/configuration.mkd

```diff
@@ -343,6 +343,20 @@ deploy:
   puppet_conf: '/opt/puppet/conf/puppet.conf'
 ```
 
+#### exclude_spec
```

Comment:
Does this need a backslash like the other settings? tbh not really sure why the others have them..
```suggestion
#### exclude\_spec
```

Reply:
I've never seen any reason to escape the `_` in mkd.